CMakeLists.txt

```diff
@@ -5,19 +5,29 @@ PROJECT(qmcpack)
 CMAKE_MINIMUM_REQUIRED(VERSION 2.8.10)
 IF(COMMAND cmake_policy)
   cmake_policy(SET CMP0003 NEW)
+  IF(CMAKE_VERSION VERSION_GREATER 3.1.3 AND NOT $ENV{CRAYPE_VERSION} MATCHES ".")
+    cmake_policy(SET CMP0056 NEW)
+    # This policy insures that the CMAKE_EXE_LINKER_FLAGS of the calling project
+    # are used in the try_compile test cmake project.  Without this
+    # clean rpath builds with mkl aren't possible. While it is unknown by
+    # cmake 2.8.10.2 on titan it still breaks the iszero test there --PWD
+  endif(NOT($ENV{CRAYPE_VERSION} MATCHES "."))
 ENDIF(COMMAND cmake_policy)
 
 
 ######################################################################
-# Version 
+# Version
 ######################################################################
 SET(QMCPACK_VERSION_MAJOR 3)
 SET(QMCPACK_VERSION_MINOR 4)
 SET(QMCPACK_VERSION_PATCH 0)
 SET(QMCPACK_VERSION "${QMCPACK_VERSION_MAJOR}.${QMCPACK_VERSION_MINOR}.${QMCPACK_VERSION_PATCH}")
 
+MESSAGE( "INITIAL CMAKE_CXX_FLAGS ${CMAKE_CXX_FLAGS}" )
+MESSAGE( "INITIAL CMAKE_CXX_FLAGS_RELEASE ${CMAKE_CXX_FLAGS_RELEASE}" )
```

Comment:
If I'm not building release, why print RELEASE. I think users should not specifically touch RELEASE.
We actually have a problem. In later lines, we have
```
MESSAGE(STATUS "CXX compiler flags: ${CMAKE_CXX_FLAGS}")
MESSAGE(STATUS "CC  compiler flags: ${CMAKE_C_FLAGS}")
```
However, this is not the full flag. RELEASE and other flags are not printed together. Do you know if there is a way to print the full flag?

Reply:
I found the layout is not elegant. If it was added for debugging. Just remove both.